03_module/simple_module/Makefile

```diff
@@ -0,0 +1,7 @@
+TARGET = msimplemodule
+obj-m += $(TARGET).o
+default:
+	$(MAKE) -C $(BUILD_KERNEL) M=$(CURDIR) modules
+
+clean:
+	$(MAKE) -C $(BUILD_KERNEL) M=$(CURDIR) clean
```

Comment:
Makefile expects `BUILD_KERNEL` being defined in environment,
but neither code, nor commit message don't explain this.